src/Packagist/WebBundle/Resources/views/user/enable_two_factor_auth.html.twig

```diff
@@ -0,0 +1,66 @@
+{% extends "PackagistWebBundle:user:packages.html.twig" %}
+
+{% import "PackagistWebBundle::macros.html.twig" as macros %}
+
+{% set isActualUser = app.user and app.user.username is same as(user.username) %}
```

Comment:
I suggest moving this inside the block (as you don't need to share it between blocks). This way, you don't need to hope that the parent template does not override this variable name before rendering the block.

Reply:
This code was copied as-is from existing templates in that same folder.  I've fixed the instances in my templates and will submit a separate PR to address the others :)